observations/forward_operators/obs_def_cice_mod.f90

```diff
@@ -175,9 +201,86 @@ end subroutine initialize_module
 
 
 !-----------------------------------------------------------------------------
-!> 
+!> This forward operator defines aggregate radar freeboard, which is 
+!> measured as the height of the ice/snow interface off the sea surface.
+
+subroutine get_expected_radar_freeboard(state_handle, ens_size, location,  &
+                                       var_sic, var_siv, var_snv, &
```

Comment:
This is in the original code, so I'll file a separate issue, but var_sic, var_siv, var_snv are unused in these freeboard routines.